docs/rpc.md

```diff
@@ -181,7 +181,7 @@ Parameters:
   zero address for regular ETH. default: regular ETH
 - `relayerEndpoint`: (optional) RPC endpoint of the relayer to use for submitting claim
   transactions.
-- `relayerCommission`: (optional) Commission in percentage that the relayer receives for
+- `relayerFee`: (optional) Fee in ETH that the relayer receives for
```

Comment:
We might want to state that a the default fee is used if not specified, to prevent any user from thinking the fee itself is optional.  :)

Does we give a clear error message if the client passes a fee that is smaller than all registered relayers are willing to accept?

Reply:
updated!

> Does we give a clear error message if the client passes a fee that is smaller than all registered relayers are willing to accept?

this will require additional features on the relayer p2p network as discussed before, will leave it for later